src/displayBackend/common/ConnectorBase.cpp

```diff
@@ -246,6 +246,28 @@ void ConnectorBase::edidPutTimings(edid* edidBlock)
 	edidPutDetailedTiming(edidBlock, 0, mCfgWidth, mCfgHeight, EDID_DPI);
 }
 
+void ConnectorBase::edidPutDisplayDescritor(edid* edidBlock)
+{
+	/*
+	 * Using second descriptor as a Display descriptor to provide display name.
+	 * 
+	 * According to documentation(VESA Enhanced EDID Standard Release A, Rev.2):
+	 * Up to 13 alphanumeric characters (using ASCII codes) may be used to define the model name
+	 * of the display product. The data shall be sequenced such that the 1st byte (ASCII code) = the 1st character,
+	 * the 2nd byte (ASCII code) = the 2nd character, etc. If there are less than 13 characters in the string,
+	 * then terminate the display product name string with ASCII code ‘0Ah’ (line feed) and pad the unused bytes
+	 * in the field with ASCII code ‘20h’ (space). 
+	 */
+	const char* display_name = "pv-display";
+	detailed_non_pixel* descriptor = &edidBlock->detailed_timings[1].data.other_data;
+	descriptor->type = 0xfc;
```

Comment:
Do not [hardcode](https://github.com/xen-troops/displ_be/blob/master/src/displayBackend/common/drm_edid.h#L159)

Reply:
Ok